include/zephyr/drivers/adc/current_sense_amplifier.h

```diff
@@ -18,6 +18,7 @@ struct current_sense_amplifier_dt_spec {
 	uint16_t sense_gain_div;
 	uint16_t noise_threshold;
 	bool enable_calibration;
+	int16_t zero_current_voltage_mv;
```

Comment:
Its pedantic, but keeping the structs more or less ordered by size helps reduce random padding bytes.
It would be good if you could move this above the boolean.